crates/sui-config/src/node.rs

```diff
@@ -51,6 +51,10 @@ pub struct NodeConfig {
     #[serde(default)]
     pub enable_reconfig: bool,
 
+    /// The duration of a checkpoint, in seconds.
```

Comment:
Should we state in the doc / `--help` that this is not to be changed? In face can we hide this option so that operators are not confused and try to set it?